gdal/frmts/mrsid/mrsiddataset.cpp

```diff
@@ -3610,6 +3610,8 @@ void GDALRegister_MrSID()
         /* for JP2CreateCopy() */
     poDriver->SetMetadataItem( GDAL_DCAP_VIRTUALIO, "YES" );
 #endif
+    poDriver->SetMetadataItem( GDAL_DCAP_COORDINATE_EPOCH, "YES" );
```

Comment:
SID also is missing from coordinate_epoch.rst

Reply:
Also, this deals with JP2MrSID, not SID.